src/lib/components/schedule/Schedule.svelte

```diff
@@ -1,17 +1,413 @@
+<script lang="ts">
```

Comment:
This is the meat

Reply:
Focusing on the kiosk/tv view which kicks in at 1920w, can you adjust the rows so we can fit details for eight classrooms without vertical scrolling?

Reply:
@philsinatra Updated